src/koffi/src/ffi.cc

```diff
@@ -1664,7 +1664,7 @@ static Napi::Value DecodeValue(const Napi::CallbackInfo &info)
 
     Napi::External<void> external = info[0].As<Napi::External<void>>();
     int64_t offset = has_offset ? info[1].As<Napi::Number>().Int64Value() : 0;
-    const uint8_t *ptr = (const uint8_t *)external.Data() + offset;
```

Comment:
This does not work, you need to cast to the right type first. (Here the compiler does not know how many bytes to skip)